stat/distuv/interfaces.go

```diff
@@ -4,19 +4,23 @@
 
 package distuv
 
+// LogProber calculates the log-probability for a given value of a random variable.
 type LogProber interface {
 	LogProb(float64) float64
 }
 
+// Rander samples a random variable.
 type Rander interface {
 	Rand() float64
 }
 
+// RandLogProber is a Rander and a LogProber.
 type RandLogProber interface {
 	Rander
 	LogProber
 }
 
+// Quantiler calculates the quantile of a distribution for given probability.
 type Quantiler interface {
 	Quantile(p float64) float64
```

Comment:
```
// Quantile returns inverse of the cumulative distribution
// function for the probability p.
```

Reply:
I'll use the new wording which we agreed for Bernoulli.

Reply:
Can you wrap the text more tightly; godoc will format text in non-code chunk doc comments, but does not do this for comments in struct or interface definitions, so we need to be more careful there. Something on the order of 50 characters is pretty good.

Reply:
Done.